BuberBreakfast/Program.cs

```diff
@@ -8,6 +12,12 @@
 
     // specify such that within a single request, whenever IBreakfastService is called, instantiate BreakfastService
     builder.Services.AddScoped<IBreakfastService, BreakfastService>();
+    builder.Services.AddScoped<IShopService, ShopService>();
+
+    builder.Services.AddDbContext<DataContext>(options =>
```

Comment:
# Remark
Dependency injection for db

# Painful lesson
Must use `options.UseMySQL` instead of `options.UseSQL`